documentation/modules/configuring/snip-reassign-partitions.adoc

```diff
@@ -1,78 +1,65 @@
-. Copy the `_reassignment.json_` file to the broker pod on which you will later execute the commands:
-+
-[source,shell,subs=+quotes]
-----
-cat _reassignment.json_ | \
-  kubectl exec _broker-pod_ -c kafka -i -- /bin/bash -c \
-  'cat > /tmp/reassignment.json'
-----
-+
-For example:
+
+. If you haven't done so, xref:proc-generating-reassignment-json-files-{context}[run an interactive pod container to generate a reassignment JSON file] named `reassignment.json`.
+
+. Copy the `_reassignment.json_` file to the interactive pod container.
 +
 [source,shell,subs=+quotes]
 ----
-cat _reassignment.json_ | \
-  kubectl exec my-cluster-kafka-0 -c kafka -i -- /bin/bash -c \
-  'cat > /tmp/reassignment.json'
+kubectl cp reassignment.json kafka-client-shell:/tmp/reassignment.json
 ----
 
-. Execute the partition reassignment using the `kafka-reassign-partitions.sh` command line tool from the same broker pod.
+. Run the partition reassignment using the `kafka-reassign-partitions.sh` script from the interactive pod container.
 +
 [source,shell,subs=+quotes]
 ----
-kubectl exec _broker-pod_ -c kafka -it -- \
-  bin/kafka-reassign-partitions.sh --bootstrap-server localhost:9092 \
-  --reassignment-json-file /tmp/reassignment.json \
-  --execute
+bin/kafka-reassign-partitions.sh --bootstrap-server
+ _<cluster_name>_-kafka-bootstrap:9093 --command-config \
```

Comment:
The command is something like ---> `bin/kafka-reassign-partitions.sh --bootstrap-server my-cluster-kafka-bootstrap:9093 --command-config /tmp/config.properties --reassignment-json-file /tmp/reassignment.json --execute```.  i guess ` /tmp/config.properties` this part got omitted in the docs. Sorry for that I have updated the document now with this edit.

Reply:
I didn't spot that. Updated now.